english/forest/fire/woods/woods.md

```diff
@@ -11,3 +11,5 @@ they are welcome to join the tribe of ancient warlocks.
 
 Maybe you decided to follow the shawman deeper into the forest?
 
+[Follow the shawman](follow/follow.md)
+[Go back to camp](camp/camp.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.